README.rst

```diff
@@ -73,6 +73,23 @@ Figures
 
 Code for generating the plots included in some of the publications using ``paltas`` can be found under the corresponding arxiv number in the ``notebooks/papers/`` folder.
 
+Contributing
+------------
+
+We welcome improvements via GitHub pull request! The workflow for contributing to ``paltas`` is the same as it is for ``lenstronomy``, please see the `lenstronomy CONTRIBUTING guidelines <https://github.com/lenstronomy/lenstronomy/blob/main/CONTRIBUTING.rst>`_. for the rundown.
+
+In brief, to contribute to ``paltas`` you will need to fork the repo on GitHub, clone it, and pip install with editable option, like this:
+
+.. code-block:: bash
+
+    $ git clone git@github.com:<your-account>/paltas.git
+    $ cd paltas
+    $ pip install -e .
+
+You should then work in a local development branch, while keeping your ``main`` branch up to date with the base repo on GitHub. (That means you'll need to ``git remote add base https://github.com/swagnercarena/lenstronomy.git`` so that you can ``git pull base main`` from your ``main`` branch).
+
+See the `lenstronomy CONTRIBUTING guidelines <https://github.com/lenstronomy/lenstronomy/blob/main/CONTRIBUTING.rst>`_. for help with python style, docstring formats, creating pull requests, and so on. We recommend that you start your pull request (PR) when you _begin_ work on a new feature or bug fix, so that the rest of the ``paltas`` community can see what you are doing and coordinate with you - when your improvement is ready you can request code review from others via the PR interface.
```

Comment:
I think this is mostly correct, besides the fact that our standard is to use Args instead of Parameters in our docstrings. Until we write our own contribution description (which we can hold off on if / until we have a larger community using paltas) this looks good.

Reply:
Regarding args vs parameters, do we follow [Google docstring style](https://www.datacamp.com/tutorial/docstrings-python)? I can add a short statement with a pointer to the right style guide.